### ISSUE TITLE
Hardcoded Origin

### ISSUE THREAD

**Issue by antinodes** [ROGUE-JCTD/Grout#3]
Currently, there is no way to specify the origin of the tileset.  This assumes that the tileset origin is the same as the SRS/CRS origin.

Should we allow for overriding the origin of the tileset?  Many of the tile services allow setting a different origin than the SRS/CRS.

**Comment by JStuhr** [comment]
If other services do it then we should too. Try and keep them similar across the board so they are easy to use and get into but have functionality that people are use to having.

**Comment by pepijnve** [comment]
Isn't the origin in one of the corners of the bounding box? If the origin is the origin of the SRS then does this imply you have negative tile coordinates for SRSs that allow negative easting/northing values.

**Comment by antinodes** [comment]
@pepijnve 

Yes, right now you can have negative tile coordinates.  The math for converting bounding box into tile indices should still work, unless I made a mistake in my math.

On the plus side however, this means you can add data to your tileset that expands the bounding box and it doesn't change your tile indices.  If you did that though you would want to increment the version number.
